run-checks.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
 
-result=$(find . -type f -name "*.tex" | xargs -n1 ./find_white_in_terminal.py)
-
 ec=0
 
+result=$(find . -type f -name "*.tex" | xargs -n1 ./find_white_in_terminal.py)
 if [[ $result ]]; then
     echo "Found one or more LaTeX commands in prompt environment, please close them with {}"
     echo "$result"
     ec=1
 fi
 
+# Style guide can use \verb to say it's not okay to use \verb
+result=$(grep -R --include=\*.tex '\\verb[^\w]' | grep -v style-guide)
```

Comment:
@redfast00 Usage of `grep` is `grep [OPTIONS] PATTERN [FILE...]`, and `[^\w]` doesn't do what you think it does (I think)?